src/python/gudhi/wasserstein/wasserstein.py

```diff
@@ -203,7 +203,9 @@ def _warn_infty(matching):
     `matching=True`) about the returned matching being `None`.
     '''
     if matching:
-        warnings.warn('Cardinality of essential parts differs. Distance (cost) is +inf, and the returned matching is None.')
+        warnings.warn('''Cardinality of essential parts differs. Distance (cost) is +inf, and the returned matching is
+                      None.
+                      ''')
```

Comment:
I don't think this split does the right thing.

Reply:
Ah yes the warning was on 2 separate lines.
Instead I factorized the beginning of the message in a string to reduce the line length on [347a4c1](https://github.com/GUDHI/gudhi-devel/pull/1034/commits/347a4c166cbf057d9c52a66997c5507c63368149).
I agree, line length changes are not the most interesting, but this is nice to stick with a limited number of characters per line as, with some IDE, you need to scroll to see the code.